AcademyApp/iOSNights2019/Scenes/Lectures/LecturesViewController.swift

```diff
@@ -85,8 +85,12 @@ extension LecturesViewController: UITableViewDelegate {
         showDetail(for: lecture)
     }
 
-    func tableView(_: UITableView, viewForHeaderInSection _: Int) -> UIView? {
-        // TODO: Dequeue
-        return nil
+    func tableView(_ tableView: UITableView, viewForHeaderInSection _: Int) -> UIView? {
+        let cell = tableView.dequeueReusableHeaderFooterView(withIdentifier: "Header")
+        return cell
+    }
+
+    func tableView(_: UITableView, heightForHeaderInSection _: Int) -> CGFloat {
+        return 70.0
```

Comment:
Do not have magic constants in a code. Rather define a variable for header height as in the case of cell height